components/GlobalPlayer.tsx

```diff
@@ -1,24 +1,15 @@
 'use client';
 import { usePlayer } from '@/providers/audio/PlayerProvider';
 import Image from 'next/image';
-import { MdPauseCircle, MdPlayCircle } from 'react-icons/md';
-import { Button } from '@/components/ui/button';
 import Scrubber from '@/components/Scrubber';
+import PlayerButtons from './PlayerButtons/PlayerButtons';
```

Comment:
please add an `index.tsx` file in `/PlayerButtons` to avoid nested import
```suggestion
import PlayerButtons from './PlayerButtons';
```